### ISSUE TITLE
Selecting Multiple files with regex on Windows doesn't works!

### ISSUE THREAD

**Issue by WiVi71** [stedolan/jq#2219]
**Describe the bug**

![0](https://user-images.githubusercontent.com/59140461/101192070-dec2b600-369d-11eb-9ffd-a223a08a48f6.gif)
Selecting multiple files with regex(like  *,?) occurs error on windows.

**To Reproduce**
```
echo {"key":"value"}> f1.json
echo {"key":"value"}> f2.json
jq . f*.json
```

**Expected behavior**
Jq should select files matches regular expression pattern.

**Error log**
```
Assertion failed!

Program: C:\Users\USER\jq.exe
File: src/main.c, Line 256

Expression: wargc == argc
```

**Environment (please complete the following information):**
 - Microsoft Windows [Version 10.0.18363.592]
 - jq-1.6(x64)

**Comment by pkoppstein** [comment]
You forgot to quote the JSON strings.
```
$ echo '{"key":"value"}' > f1.json
$ echo '{"key":"value"}' > f2.json

$ jq . f*.json
{
  "key": "value"
}
{
  "key": "value"
}
$
```

**Comment by WiVi71** [comment]
echo on windows terminal doesn't requires quotes.
Also, This bug is not affected by the format of the file.

```
#>echo( >f1.json

#>echo( >f2.json

#>jq . f*.json
Assertion failed!

Program: C:\ProgramData\chocolatey\lib\jq\tools\jq.exe
File: src/main.c, Line 256

Expression: wargc == argc

#>
```

**Comment by WiVi71** [comment]
This bug was fixed on pull #1696. It's currently available on master.
